core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -32,52 +32,35 @@
       <p>title: <[collection.title]></p>
       <p>category: <[collection.category]></p>
       <p>objective: <[collection.objective]></p>
+      <p>version: <em><[collection.version]></em></p>
+      <a ng-href="/collection/<[collection.id]>" target="_blank">Go to Collection</a>
 
-      <!-- Examples for modifying the collection. This is placeholder and should not go into develop. -->
-      <hr>
-
-      <h3>Current collection</h3>
-      <strong>Details</strong>
-      <br>
-      Title: <em><[collection.title]></em>
-      <br>
-      Category: <em><[collection.category]></em>
-      <br>
-      Objective: <em><[collection.objective]></em>
-      <br>
-      Version: <em><[collection.version]></em>
+      <!-- List of acquired and pre-requisite skills !-->
       <br>
       <br>
+      <strong>Skills</strong>
+      <collection-skills-list-directive collection-skills="collectionSkills">
+      </collection-skills-list-directive>
+
 
+      <!-- List of explorations !-->
+      <br>
       <strong>Explorations</strong>
       <ul>
         <li ng-repeat="node in collection.nodes">
-          <div>
-            [<[node.exploration.id]>] <em><[node.exploration.title]></em>
-            <br>
-            <div ng-if="node.prerequisite_skills.length > 0">
-              Prerequisite skills:
-              <ul>
-                <li ng-repeat="skill in node.prerequisite_skills"><em><[skill]></em></li>
-              </ul>
-            </div>
-            <div ng-if="node.acquired_skills.length > 0">
-              Acquired skills:
-              <ul>
-                <li ng-repeat="skill in node.acquired_skills"><em><[skill]></em></li>
-              </ul>
-            </div>
-          </div>
+          <collection-node-directive
+            prerequisite-skills="node.prerequisite_skills" acquired-skills="node.acquired_skills"
+            exploration-id="node.exploration.id" exploration-title="node.exploration.title"
+            add-change="addChange(change)"
+            delete-exploration="deleteExploration(expId)"></collection-node-directive>
         </li>
       </ul>
 
-      <h3>Examples to change the collection<span ng-if="changeList.length > 0"> <em>(<[changeList.length]> changes pending)</em></h3>
-
-      <a ng-href="/collection/<[collection.id]>" target="_blank">Go to Collection</a>
+      <!-- Edit options for major collection properties !-->
       <br>
-      <input ng-model="newExpId" type="text" placeholder="New exp ID" /><button ng-click="addExploration(newExpId)">Add Exploration</button>
+      <strong>Edit Collection</strong>
```

Comment:
Perhaps this section should be a directive, also. That will thin out the final aspects of the collection editor which has to do with changes (except the part stores the change list and sends it to the backend). WDYT?